src/rgw/driver/sfs/sqlite/sqlite_buckets.cc

```diff
@@ -42,6 +45,22 @@ std::optional<DBOPBucketInfo> SQLiteBuckets::get_bucket(
   return ret_value;
 }
 
+std::optional<std::pair<std::string, std::string>> SQLiteBuckets::get_owner(
+    const std::string& bucket_id
+) const {
+  auto storage = conn->get_storage();
+  const auto rows = storage.select(
+      columns(&DBUser::user_id, &DBUser::display_name),
+      inner_join<DBUser>(on(is_equal(&DBBucket::owner_id, &DBUser::user_id))),
+      where(is_equal(&DBBucket::bucket_id, bucket_id))
+  );
+  if (rows.size() == 0) {
+    return std::nullopt;
+  }
+  const auto row = rows[0];
+  return std::make_pair(std::get<0>(row), std::get<1>(row).value());
```

Comment:
feels like the either the first `get<>()` is missing a `.value()`, or the `.value()` on the second `get<>()` should not be there.

Reply:
Thanks for pointing this out. It is actually neither. The second is an optional and value() should have been a value_or("")